unit1-wrap-up/activity-mind-map.md

```diff
@@ -0,0 +1,13 @@
+# Unit 1 - Reflection Activity
+
```

Comment:
Nitpicking: I know this is goofy, but I'm suggesting a template header.

```suggestion

## Goals

```

Reply:
incorporated.